hw1/Sem01Task04.py

```diff
@@ -0,0 +1,24 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов.
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое
+# количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# спрашиваем сколько журавликов сделади дети
+birds = int(input("Сколько всего журавликов сделали дети?: "))
+
+
+# Уравнение для решения задачи:
+# petya, sergo, katya
+# petya = sergo
+# katya = 2*(petya+sergo) = 4*petya
+# birds = katya + petya + sergo = 4*petya + petya + petya = 6*petya
+# petya = birds/6
+
+# вычисляем сколько журавликов сделал Петя округляем, чтоб не получилось 1,5 землекопа
+petya = int(birds/6)
```

Comment:
Во-первых, это можно сделать через оператор целочисленного деления
```suggestion
petya = birds // 6
```
Во-вторых, не плохо было бы в начале проверить именно входные данные кратность `6`.